exampleLitActions/src/conditionalSigning.ts

```diff
@@ -0,0 +1,238 @@
+      capabilityAuthSigs: [capacityDelegationAuthSig],
+      expiration: new Date(Date.now() + 1000 * 60 * 60 * 24).toISOString(), // 24 hours
+      resourceAbilityRequests: [
+        {
+          resource: new LitActionResource("*"),
+          ability: LIT_ABILITY.LitActionExecution,
+        },
+      ],
+      authNeededCallback: async ({ resourceAbilityRequests, expiration, uri }) => {
+        const toSign = await createSiweMessage({
+          uri: uri || "",
+          expiration: expiration || "",
+          resources: resourceAbilityRequests || [],
+          walletAddress: await ethersWallet.getAddress(),
+          nonce: await litNodeClient.getLatestBlockhash(),
+          litNodeClient: litNodeClient,
+        });
+
+        return await generateAuthSig({
+          signer: ethersWallet,
+          toSign,
+        });
+      },
+    });
+
+    // Create auth signature for the action
+    const authSig = await (async () => {
+      const toSign = await createSiweMessage({
+        uri: "http://localhost",
+        expiration: new Date(Date.now() + 1000 * 60 * 60 * 24).toISOString(), // 24 hours
+        walletAddress: await ethersWallet.getAddress(),
+        nonce: await litNodeClient.getLatestBlockhash(),
+        resources: [
+          {
+            resource: new LitActionResource("*"),
+            ability: LIT_ABILITY.LitActionExecution,
+          },
+        ],
+        litNodeClient: litNodeClient,
+      });
+      return await generateAuthSig({
+        signer: ethersWallet,
+        toSign,
+      });
+    })();
+
+    // Data to sign - using a known hash for consistent testing
+    const dataToSign = ethers.utils.arrayify(
+      ethers.utils.keccak256([1, 2, 3, 4, 5])
+    );
+    
+    // Execute the Lit Action
+    console.log("🔄 Executing Lit Action with conditional signing...");
+    const litActionResult = await litNodeClient.executeJs({
+      sessionSigs: sessionSigs,
+      code: litActionCode,
+      jsParams: {
+        conditions: [
+          {
+            conditionType: "evmBasic",
+            contractAddress: "",
+            standardContractType: "",
+            chain: CHAIN_TO_CHECK_CONDITION_ON,
+            method: "eth_getBalance",
+            parameters: [":userAddress", "latest"],
+            returnValueTest: {
+              comparator: ">=",
+              value: "0",
+            },
+          },
+        ],
+        authSig: authSig,
+        chain: CHAIN_TO_CHECK_CONDITION_ON,
+        dataToSign: dataToSign,
+        publicKey: pkpPublicKey,
+      },
+    });
+    
+    console.log("✅ Lit Action executed successfully");
+    console.log(JSON.stringify(litActionResult, null, 2));
+
+    return {
+      success: litActionResult.success,
+      signatureResult: litActionResult.signatures || {},
+      response: typeof litActionResult.response === 'string' 
+        ? litActionResult.response 
+        : JSON.stringify(litActionResult.response)
+    };
```

Comment:
So we should just return `litActionResult` here, or just `return litNodeClient.executeJs(...`